website/static/custom.css

```diff
@@ -118,6 +136,15 @@ a .softkey, a .key {
   text-transform: uppercase;
 }
 
+.annotate {
+  border-radius: 5px;
+  padding: 1px 4px;
+  border-color: rgba(216, 0, 41, 0.8);
+  color: rgba(216, 0, 41, 0.8);
```

Comment:
Ideally these could be solid colours (not using opacity) so when they're used in things with backgrounds, like blockquotes, they stay the same colour for consistency with the others.

Reply:
Could be solid colour #D80029, this is the colour used in the images